host/xtest/asym_cipher_perf.c

```diff
@@ -0,0 +1,747 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2023, Huawei Technologies Co., Ltd
+ * All rights reserved.
+ */
+
+#include <fcntl.h>
+#include <math.h>
+#include <stdint.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <strings.h>
+#include <sys/ioctl.h>
+#include <sys/mman.h>
+#include <sys/stat.h>
+#include <sys/types.h>
+#include <ta_asym_cipher_perf.h>
+#include <tee_client_api.h>
+#include <tee_client_api_extensions.h>
+#include <time.h>
+#include <unistd.h>
+
+#include "crypto_common.h"
+#include "xtest_helpers.h"
+#include <utee_defines.h>
+
+static TEEC_Context ctx;
+static TEEC_Session sess;
+
+static TEEC_SharedMemory in_shm = {
+	.flags = TEEC_MEM_INPUT | TEEC_MEM_OUTPUT
+};
+static TEEC_SharedMemory out_shm = {
+	.flags = TEEC_MEM_INPUT | TEEC_MEM_OUTPUT
+};
+static TEEC_SharedMemory hash_shm = {
+	.flags = TEEC_MEM_INPUT | TEEC_MEM_OUTPUT
+};
+
+static void errx(const char *msg, TEEC_Result res, uint32_t *orig)

```

Comment:
It seems that there's quite a bit of copy & paste from `host/xtest/hash_perf.c` or `host/xtest/aes_perf.c` in this file.
Duplicated code is one thing, but now we have a third copy. It's time to consolidate the code before it get even harder to maintain.

Reply:
I agree.

Reply:
OK.I will consilidate them.What is the new TA's name?perf?

Reply:
I will do that with another commit after the asyn_cipher one. Is that OK?

Reply:
> OK.I will consilidate them.What is the new TA's name?perf?

I suggest crypto_perf.